pxd.h

```diff
@@ -159,6 +161,25 @@ struct pxd_add_ext_out {
 	struct pxd_update_path_out paths; /**< backing device paths */
 };
 
+/* Device capability flags */
+#define PXD_CAP_WRITE_ZEROES    (1 << 0)  /* WriteZero→Discard optimization */
```

Comment:
nit.
this is a per device capability for write zeroes. Can you name it accordingly.
PXD_DEV_CAP_WRITE_ZEROES.